quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -1116,8 +1116,6 @@ function ziti_createEnvFile {
   export ZITI_SIGNING_ROOTCA_NAME="${ZITI_SIGNING_CERT_NAME}-root-ca"
   export ZITI_SIGNING_INTERMEDIATE_NAME="${ZITI_SIGNING_CERT_NAME}-intermediate"
 
-  export ZITI_BIN_ROOT="${ZITI_HOME}/ziti-bin"
```

Comment:
In `generateEnvFile` there is already a call to check if ZITI_BIN_ROOT is set, if not then it sets it, otherwise it leaves it alone. This just overwrites it every time.